python/pyspark/sql/dataframe.py

```diff
@@ -1597,11 +1607,14 @@ def toDF(self, *cols):
         return DataFrame(jdf, self.sql_ctx)
 
     @since(1.3)
-    def toPandas(self):
+    def toPandas(self, useArrow=False):
         """Returns the contents of this :class:`DataFrame` as Pandas ``pandas.DataFrame``.
 
         This is only available if Pandas is installed and available.
 
+        :param useArrow: Make use of Apache Arrow for conversion, pyarrow must be installed
```

Comment:
For now we should probably have this as experimental, and also make clear what subset of data types are supported here (or do we expect to support all of the data types)?